src/app/personsok/personsok-utils.ts

```diff
@@ -19,6 +22,96 @@ export type PersonSokFormState = {
     kjonn: string;
     _minimumskrav: string;
 };
+export const useFormstate = formstateFactory<PersonSokFormState>(values => {
+    let fornavn = undefined;
```

Comment:
Om du ønsker å beholde mange av de testene som finnes så kan selve funksjonen her trekkes, e.g;
```
export function validatorPersonsok(values: PersonSokFormState) { ... }
export const useFormstate = formstateFactory<PersonSokFormState>(validatorPersonsok);
```

Reply:
Det var sånn det var - men så at det ble gjort som dette andre steder vi bruker useFormstate så da ønsket jeg å gjøre det likt. Men kanskje ikke en god måte?

Reply:
Er mulig å få til, men testing av selve validatoren må da endres ganske mye siden vi ikke lengre har tilgang til selve validator-funksjonen.

Så da må vi evt teste `useFormstate` hooken som blir laget. Det er sånn testene til use-formstate er laget, så kan ta en titt på de for å se oppsettet om du ønsker å gå i den retningen; https://github.com/nutgaard/use-formstate/blob/master/test/use-formstate.test.ts#L36